core/src/main/java/jenkins/util/ResourceBundleUtil.java

```diff
@@ -70,55 +66,20 @@ private ResourceBundleUtil() {
      * @throws MissingResourceException Missing resource bundle.
      */
     public static @NonNull JSONObject getBundle(@NonNull String baseName, @NonNull Locale locale) throws MissingResourceException {
-        String bundleKey = baseName + ":" + locale;
-        JSONObject bundleJSON = bundles.get(bundleKey);
+        var bundleKey = baseName + ":" + locale;
```

Comment:
I guess there is no style policy in core beyond what little Checkstyle enforces, but is there any precedent for using `var`?